### ISSUE TITLE
Should stringTest skip voicing strings?

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/chipper#1198]
This came up when stringTest replaced a voicing string and caused code to fail, see https://github.com/phetsims/sun/issues/739.

The purpose of stringTest is to test layout. If voicing strings are not displayed, then should voicing strings be skipped by stringTest? If they were in a standard place in the JSON, it would be relatively easy to skip them.

**Comment by pixelzoom** [comment]
> Any other solution I can think of would require many refactorings to put strings into sections based on visual/non-visual. Having added many complex strings to the strings files for these complicated features (description and voicing), I much prefer organization based on content. I have found a lot of description and voicing strings to be similar, and I would prefer to keep them together.

While I appreciate the overlap in visual/non-visual strings... Perhaps this is a case where it's appropriate to put the burden on translators and have some duplication that benefits easy-of-programming.  That might be preferable to solutions like https://github.com/phetsims/sun/commit/0ffa150f9eeeadd6fd4ce710b1b6bbde2015c4b8 that involve checking `stringTest` -- and having to remember to do so:

```diff
+    // Don't test the contents of strings when ?stringTest is enabled
+     assert && assert( !!phet.chipper.queryParameters.stringTest ||
+                       options.comboBoxVoicingNameResponsePattern.includes( '{{value}}' ),
```

**Comment by marlitas** [comment]
Meeting 12/4
- Voicing will be getting a lot of work within the next few months.
- We imagine this will come up again and address as that happens.
